brave/brave-core
# Add setting options to toggle NTP widgets

Spec: https://github.com/brave/brave-browser/issues/4510 (Current scope of this PR does not include right clicking on the widgets to open up a mini-menu)
Brave-ui Sister PR: https://github.com/brave/brave-ui/pull/503
Quick View:
![toggle-widgets](https://user-images.githubusercontent.com/9125231/59808938-453df380-92b3-11e9-99bb-025bc9b5578d.gif)
## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
1. Open new tabs page with a fresh profile - all NTP elements should appear
2. Open settings menu from the settings icon - 3 new settings should exist below show background
3. Toggle the three options; show clock; show stats; show top sites - they should disappear/reappear on the page
4. Settings should persist after browser is closed and reopened

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
